src/sessions/base-session-handler.ts

```diff
@@ -406,10 +415,33 @@ export default abstract class BaseSessionHandler {
     await Promise.all(promises);
   }
 
+  async waitForSessionConnected (session: IExtendedMediaSession): Promise<undefined> {
```

Comment:
`Promise<undefined>` is that right?

Reply:
I guess it would be since if the `connectionState` is 'connected' then we resolve to undefined but it just seems odd to me I guess